CFC_WebApp/api/cfc_webapp.py

```diff
@@ -382,7 +401,10 @@ def getUUID(request):
     retUUID = user_uuid
     logging.debug("skipAuth = %s, returning fake UUID %s" % (skipAuth, user_uuid))
   else:
-    userToken = request.json['user']
+    if inHeader:
+      userToken = request.headers.get('Authorization').split()[1]
```

Comment:
Is the header called "Authorization" or "User"? It looks like you assume it to be "User" in `getCarbonCompare()` and in the client code, but you assume it is "Authorization" here.